### PR TITLE
remove staff view customlabels; Closes #1330

### LINKED ISSUE #1330: White labelling for Announcements, Students, and Badges
Add white labelling to several features, to allow deck owners/staff to provide custom labels for features.  

Currently, white labelling is available for Tags and Groups, via SiteConfig form:

image

We need to add more:

- [ ] Announcements
- [ ] Students
- [ ] Badges

Also:

- [ ] These white labelling options should be grouped together under a common heading in the SiteConfig form.  Perhaps a collapsed thing like we do for the advanced section of quests?

### PR DESCRIPTION
**Pull request by Dylan-RG-Patterson**
### What?
This PR Removes/Reverts all non-student facing instances of custom labels that are set with logic instead of template tags.

### Why?
Label customization requires extensive testing and logic to ensure reliability across every instance of label words used across the entire site, limiting their implementation reduces code bloat.

### How?
Unnecessary labels have been removed, and any associated tests have been rewritten or deleted to only test for used customlabels.

### Testing?
No new tests have been added, tests for reverted custom label instances have been removed.

### Screenshots (if front end is affected)
non-staff frontend is unaffected.

### Review request
@tylerecouture

### CODE REVIEW

File: src/announcements/views.py [hunk 1]
```diff
@@ -194,9 +194,9 @@ def form_valid(self, form):
 
     def get_success_message(self, cleaned_data):
         if self.object.draft:
-            return f"Draft {SiteConfig.get().custom_name_for_announcement} created."
+            return "Draft Announcement created."
```
Comment (tylerecouture) [inline src/announcements/views.py:197]: Leave this one.  Doesn't need a test.

File: src/announcements/views.py [hunk 2]
```diff
@@ -194,9 +194,9 @@ def form_valid(self, form):
...
         else:
-            return f"New {SiteConfig.get().custom_name_for_announcement} published and broadcast to {SiteConfig.get().custom_name_for_student}s!"
+            return "New Announcement published and broadcast to Students!"
```
Comment (tylerecouture) [inline src/announcements/views.py:199]: Leave this one for both announcement and student.  Doesn't need a test.

File: src/announcements/views.py [hunk 3]
```diff
@@ -228,9 +228,9 @@ def get_context_data(self, **kwargs):
 
     def get_success_message(self, cleaned_data):
         if self.object.draft:
-            return f"Draft {SiteConfig.get().custom_name_for_announcement} updated."
+            return "Draft Announcement updated."
```
Comment (tylerecouture) [inline src/announcements/views.py:231]: Leave this one.  Doesn't need a test.

File: src/announcements/views.py [hunk 4]
```diff
@@ -228,9 +228,9 @@ def get_context_data(self, **kwargs):
...
         else:
-            return f"{SiteConfig.get().custom_name_for_announcement} updated but NOT (re-)broadcasted to {SiteConfig.get().custom_name_for_student}s."
+            return "Announcement updated but NOT (re-)broadcasted to students."
```
Comment (tylerecouture) [inline src/announcements/views.py:233]: Leave this one.  Doesn't need a test.

File: src/badges/views.py [hunk 5]
```diff
@@ -220,7 +220,7 @@ def bulk_assertion_create(request, badge_id=None):
         # TODO: Why does this form use Profile model instead of User model?
         profiles = form.cleaned_data['students']
 
-        result_message = f"{SiteConfig.get().custom_name_for_badge} {str(badge)} granted to "
+        result_message = "Badge {str(badge)} granted to "
         for profile in profiles:
```
Comment (tylerecouture) [inline src/badges/views.py:224]: Leave this one.  Doesn't need a test.

File: src/badges/views.py [hunk 6]
```diff
@@ -253,7 +253,7 @@ def assertion_create(request, user_id, badge_id):
     if form.is_valid():
         new_ass = form.save(commit=False)
         BadgeAssertion.objects.create_assertion(new_ass.user, new_ass.badge, transfer=new_ass.do_not_grant_xp)
-        messages.success(request, f"{SiteConfig.get().custom_name_for_badge} {str(new_ass)} granted to {str(new_ass.user)}")
+        messages.success(request, f"Badge {str(new_ass)} granted to {str(new_ass.user)}")
```
Comment (tylerecouture) [inline src/badges/views.py:256]: Leave this one.  Doesn't need a test.

**Review by tylerecouture — CHANGES_REQUESTED** [review]
See review comments.

File: src/announcements/views.py (hunk 1 above)
Comment (Dylan-RG-Patterson) [reply src/announcements/views.py:197]: Label preserved

File: src/announcements/views.py (hunk 2 above)
Comment (Dylan-RG-Patterson) [reply src/announcements/views.py:199]: label preserved

File: src/announcements/views.py (hunk 3 above)
Comment (Dylan-RG-Patterson) [reply src/announcements/views.py]: label preserved

File: src/announcements/views.py (hunk 4 above)
Comment (Dylan-RG-Patterson) [reply src/announcements/views.py]: label reserved.

File: src/badges/views.py (hunk 5 above)
Comment (Dylan-RG-Patterson) [reply src/badges/views.py]: label preserved.

File: src/badges/views.py (hunk 6 above)
Comment (Dylan-RG-Patterson) [reply src/badges/views.py]: label preserved.

**Review by tylerecouture — APPROVED** [review]

**Pull request merged**